api/pom.xml

```diff
@@ -95,7 +95,7 @@
 	    <dependency>
             <groupId>com.newrelic.agent.java</groupId>
             <artifactId>newrelic-java</artifactId>
-            <version>5.13.0</version>
+            <version>6.1.0</version>
```

Comment:
Make this a top level variable, so that one change in the parent Pom keeps NewRelic up to date.